MultiROMMgr/src/main/assets/devices.json

```diff
@@ -332,6 +332,52 @@
                     "path": "/dev/block/platform/msm_sdcc.1/by-name/cache"
                 }
             ]
+        },
+        {
+            "names": [ "sirius", "D6503", "D6502", "D6506", "D6543" ],
+            "manifest_url": "http://multirom.myself5.de/sirius/manifest.json",
+            "check_gpg": false,
+            "ubuntu_touch": {
+                "enabled": false,
+                "base_url": ""
+            },
+            "devices": [
+                {
+                    "name": "recovery",
+                    "path": "/dev/block/platform/msm_sdcc.1/by-name/FOTAKernel"
+                },
+                {
+                    "name": "boot",
+                    "path": "/dev/block/platform/msm_sdcc.1/by-name/boot"
+                },
+                {
+                    "name": "cache",
+                    "path": "/dev/block/platform/msm_sdcc.1/by-name/cache"
+                }
+            ]
+        },
+        {
+            "names": [ "z3", "D6603", "D6602", "D6633", "D6643" ],
```

Comment:
leo is not getting used by any Rom as far as I know, CM break a naming rule for the Z3 and called it z3 rather than leo. I guess it is because there is already a device called leo, the HTC HD2. So to not cause any confusion (and trouble) I'd also prefer not to add leo in case someone installs MultiROMMgr on his/her HD2 and screws it up by flashing Z3 files. (same counts for Z3C, codename aries btw)